scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RuntimeReducerEstimator.scala

```diff
@@ -44,43 +80,32 @@ object RuntimeReducerEstimator {
   def getReduceTimes(history: Seq[FlowStepHistory]): Seq[Seq[Double]] =
     history.map { h =>
       h.tasks
-        .filter { t => t.taskType == "REDUCE" && t.status == "SUCCEEDED" && t.finishTime > t.startTime }
-        .map { t => (t.finishTime - t.startTime).toDouble }
+        .filter { t => t.taskType.contains("REDUCE") && t.status.contains("SUCCEEDED") }
```

Comment:
because then I still need `finishedTime` and `startTime` and I need zip them twice.